site/_resources/deprecation-policy.md

```diff
@@ -61,3 +61,13 @@ We use the following cycle for config file settings:
 - We wait the timeline period
 - The setting is moved or removed.
 
+## Metrics
+
+Removing/Renaming metrics is a breaking change as it might cause alerts/ dashboards to break for operators.
+
+We try to use following cycle for deprecating metrics:
+- Description for metrics should be updated to mark these metrics as deprecated.
+- If you are renaming metrics, then add metrics with new names
+- Update [grafana dashboards](../../examples/grafana/02-grafana-configmap.yaml) if applicable
```

Comment:
The links to the dashboards & alertmanager aren't working. Here's the preview page: https://deploy-preview-3362--projectcontour.netlify.app/resources/deprecation-policy/

Reply:
fixed.